lib/MusicBrainz/Server/Controller/OAuth2.pm

```diff
@@ -119,6 +119,10 @@ sub token : Local Args(0)
 {
     my ($self, $c) = @_;
 
+    $c->res->header('Access-Control-Allow-Origin' => '*');
+    $self->_send_options_response($c, 'POST')
+        if $c->request->method eq 'OPTIONS';
+
     $self->_enforce_tls($c);
```

Comment:
Should this be checked first?

Reply:
Yeah, well, it should go after the `$c->res->header` line and before the `$self->_send_options_response` line. Fixed now.